di-ipv-credential-issuer-stub/src/main/java/uk/gov/di/ipv/stub/cred/handlers/AuthorizeHandler.java

```diff
@@ -357,6 +360,26 @@ public AuthorizeHandler(
                         HTTPResponse httpResponse = httpRequest.send();
                     }
 
+                    if (F2F_SEND_VC_QUEUE && F2F_SEND_ERROR_QUEUE) {
```

Comment:
Thinking should it just be if `F2F_SEND_ERROR_QUEUE` checkbox is checked not both?